ph5/core/pmonitor.py

```diff
@@ -26,14 +26,16 @@
 TIMEOUT = 500 * 4
 
 try:
-    from PySide import QtGui, QtCore
-except Exception as e:
-    LOGGER.error("No PySide: {0}".format(e.message))
-    try:
-        from PyQt4 import QtGui, QtCore
-        QtCore.Signal = QtCore.pyqtSignal
-    except Exception:
-        LOGGER.error("PyQt4 or PySide required: {0}".format(e.message))
+    from PyQt4 import QtGui, QtCore
```

Comment:
I don't think this import should change.

Reply:
Sure, I'll work on this now.